### PR TITLE
Book page and announcement blog post

### PR DESCRIPTION
**Pull request by mosabua**
@martint @mattsfuller this essentially the same content as on the OReilly site. I am working on a blog post for this site in a separate PR.

Not sure about which Buy links we should have .. we decided on OReilly only.

Added link to code repo
Added blog post announcement

To be published next week when free copies are available.

### CODE REVIEW

**Comment by martint** [comment]
Can you squash the commits?

**Comment by mosabua** [comment]
Squashed

**Comment by mosabua** [comment]
Squashed now @martint  .. also let me know if you want a screenshot or demo of what it looks like

File: community.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ any issue you find with Presto.
 
 ## External Resources
 
+### Books
+
+* [Presto: The Definitive Guide](./presto-the-definitive-guide.html)
+
```
Comment (martint) [inline community.md:36]: We may also want to add it to the resources page.

**Comment by martint** [comment]
> let me know if you want a screenshot or demo of what it looks like

No need. The netlify action produces a preview: https://deploy-preview-75--prestosql.netlify.com/presto-the-definitive-guide.html

File: community.md (hunk 1 above)
Comment (mosabua) [reply community.md:36]: the resources page is kinda odd .. its only libraries and such .. I added it here after chatting with @electrum a while ago .. but I can add something there as well if you like ..

Comment (martint) [reply community.md:36]: No, that's ok. Let's keep it as is. We can change it or add it later.

File: _posts/2020-04-11-the-definitive-guide.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+The first part of the book establishes what Presto is, and gets you quick wins
+to install a minimal setup, run it, connect to it with the CLI and an
+application using the JDBC driver and run some SQL queries. 
+
+The second part dives into the details of the Presto architecture, query
+planning, connectors for all sorts of data sources and SQL usage. There is a lot
+to learn and digest in these main sections.
+
+In the third part we are rounding things out with tuning tips, a good overview
```
Comment (martint) [inline _posts/2020-04-11-the-definitive-guide.md:42]: "we round things out"

File: _posts/2020-04-11-the-definitive-guide.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+of the Web UI, usage of other tools, security configuration and more tips to get
+Presto into production.
+
+Of course, putting all this information together requires work from many people.
+And in fact we did get lots of help from members of the Presto community and
+O'Reilly.
+
+Specifically we have some great news from our major supporter, Starburst!
```
Comment (martint) [inline _posts/2020-04-11-the-definitive-guide.md:50]: Missing comma after "Specifically"

File: _posts/2020-04-11-the-definitive-guide.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+<!--more-->
+
+<img src="../../../../assets/ptdg-cover.jpg" align="right" width="400">
+
+With the help of O'Reilly, the book is now available in digital form, and paper
+copies are just around the corner as well. You can find more information about
+the book on [our permanent page about
+it](../../../../presto-the-definitive-guide.html).
+
+It is based on the very recent 330 release of Presto and broken up into three
```
Comment (mattsfuller) [inline _posts/2020-04-11-the-definitive-guide.md:28]: Perhaps mention somewhere that although based on 330, it's not really version-specific. Not sure how we'd want to phrase that.

Comment (mosabua) [reply _posts/2020-04-11-the-definitive-guide.md:28]: Fixed

**Pull request merged**